DBW Updates w/o PID

srnimani opened the pull request:
Updated the DBW-node to include 'Twist Controller' object and subscription to topics of interest. Added control logic for Steer, Break and Velocity..

srnimani committed:
DBW Updates w/o PID

eqbal committed:
Minor styling suggestions related to DBW node

eqbal reviewed (approved):
Thanks for sharing @srnimani. I have some minor styling suggestions that I included in https://github.com/eqbal/CarND_Capstone_CarX/pull/20. Review and if you like the changes just merge it to your branch. But overall I'll accept this PR as it looks good to go. 👍

srnimani committed:
Merge pull request #20 from eqbal/eki-dbw_styling

Minor styling suggestions related to DBW node

eqbal commented:
@srnimani all is good with me 👍 you can merge the branch if you want. Would recommend to keep it until the branch is done. If you want to sync it with the changes with other branches just rebase/merge the targeted branch to yours.

dmavridis committed:
Initial version of waypoint updater

The basic functionality of updater, reading the base_waypoints topic and publishing a shorter list of waypoints on final_waypoints topic.
Issues:
- Car not moving when applied with current dbw node.
- Issue when the points ahead exceed the length of the list? Need to cycle to the beginning?

Commit:
Added tl detector version with functional code to detect the closest waypoint to the closest traffic light

srnimani committed:
Merge remote-tracking branch 'origin/dimi-waypoint_updater' into mani-DBW
Meging with Waypoint Code

srnimani committed:
Merge remote-tracking branch 'origin/ja-tl_detector' into mani-DBW
Merging with TL detector

srnimani committed:
Fixed a comment line

srnimani committed:
Added PID Controller, untested

srnimani committed:
Uploading right files

srnimani committed:
Uploading right files

srnimani committed:
Updated PID and Yaw Controls

srnimani committed:
Merge branch 'master' of https://github.com/eqbal/CarND_Capstone_CarX into mani-DBW

srnimani committed:
Some more refinements to Twist Controller, untested

srnimani committed:
Some more bug fixes

srnimani committed:
Fixed issues related to Twist attribute

srnimani committed:
Formatting issuse fixed..

dmavridis committed:
Add files via upload

srnimani committed:
Merged DM and Mani's file

srnimani commented:
This is my latest of the DBW Node update.. Merge with master if found good..
